### DISCUSSION TITLE
Docker & kubernetes support in the upcoming days

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [reactplay/react-play#301] [Ideas]
Thinking of containerizing the reactplay project using docker and make it available on docker hub. Thinking of only containerizing the local environment for now.  What do you say @atapas sir

**Comment by atapas** [comment]
@hasnainmakada-99 Basic docker support is already in. We have the config files placed too. Try using it and let me know.

**Reply by hasnainmakada-99** [reply]
Sure tapas sir
